README.md

```diff
@@ -1,5 +1,71 @@
-# metadata-filter ![Test][WorkflowBadge]
+# Metadata Filter ![Test][WorkflowBadge] ![NPM][NpmBadge]
 
 A module for cleaning up artist, album, and song names.
 
+## Installation
+```
+npm i metadata-filter
+```
+
+## Usage
+
+### Single Filters
+You can call MetadataFilter's static filter functions for basic, one-line filter functionality.
+Functions available: `youtube`, `removeRemastered`, `removeVersion`, `removeLive`, `removeCleanExplicit`, `albumArtistFromArtist`, `normalizeFeature`, `removeFeature`, `fixTrackSuffix`, `filterWithFilterRules`
+
+```javascript
+const MetadataFilter = require('metadata-filter');
+
+console.log(MetadataFilter.youtube('Car Bomb - Scattered Sprites (Official Music Video)')); // Car Bomb - Scattered Sprites
+console.log(MetadataFilter.removeRemastered('Jane Doe (Remastered)')); // Jane Doe
+console.log(MetadataFilter.removeVersion('Daft Punk - Get Lucky (Album Version)')); // Daft Punk - Get Lucky
+```
+
+### Multiple Filters
+You can also to use multiple filters on a string at once by creating a `MetadataFilter` object which combines multiple functions from above, or by using one of the pre-existing [filter objects](#pre-defined-filter-sets).
+
+First, create a filter set. This is a set of rules for artists, albums, tracks, and albumArtists.
+```javascript
+const filterSet = {
+  track: [
+    MetadataFilter.removeRemastered,
+    MetadataFilter.fixTrackSuffix,
+    MetadataFilter.removeLive,
+  ],
+  album: [
+    MetadataFilter.removeRemastered,
+    MetadataFilter.fixTrackSuffix,
+    MetadataFilter.removeLive,
+  ],
+};
+```
+
+Then, construct a MetadataFilter using this filter set.
+```javascript
+const filter = new MetadataFilter(filterSet);
+console.log(filter.filterField('album', 'Nevermind (Remastered)')) // Nevermind
+console.log(filter.filterField('track', 'In Bloom - Nevermind Version')) // In Bloom
+```
+
+### Pre-defined Filter Sets
+There are also pre-defined filter sets available for easy access. For example, the above filter set can be acquired using `getSpotifyFilter()`:
+
+```javascript
+const filter = MetadataFilter.getSpotifyFilter();
+```
+
```

Comment:
Shall we list the other pre-defined ones as a list?

Reply:
Yep, that sounds good. I'll create an "API Reference" section.